MessengerOnFileForLinux/MessageFramework/test/StringSumTest.cpp

```diff
@@ -1,7 +1,14 @@
 #include <gtest/gtest.h>
 #include <StringSum.hpp>
 
-TEST(StringSumTest, trueIsTrue)
+TEST(StringSumTest, sumString)
```

Comment:
Ta nazwa testu niewiele mówi. Może coś w stylu isComponentsArePlacedInSquareBracketsAndReturnAsContinuous string? Nie mam na razie lepszych pomysływó, może mi coś jeszcze przyjdzie do głowy albo tobie

Reply:
Klasa nazywa się StringSum, jedyne co robi to cumuje stringi, czemu nie możemy nazwać testu, który sprawdza, że sumuje stringi sumString?

Reply:
Jak miałem jakiś czas temu szkolenie z Cpp WL, to gość mówił żeby nazwy testów dokumentowały dzialanie funkcjonalności. Czytasz nazwe testu, i wiesz bez patrzenia w implementacje tylko po samej nazwie co ona robi

Reply:
No a ja czytałem Wujka Boba, że powinna mieć jak najmniej i mówić co jest testowane zamiast długich tasiemców, ale ok ta może jest zbyt krótka i mało precyzyjna:

isStringsAreSummedWithBrackets - co sądzisz o takiej?

Reply:
ok, akceptuje